### DISCUSSION TITLE
I want to group notifications by organization/user or exclude specific organizations

### DISCUSSION THREAD

**Discussion by SRNissen** [community/community#77768] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

# AS A(N):

Employee at an organization that uses github

# I WANT TO:

Find discussions that I have participated in that are not part of my employment

# SO THAT:

I can re-read old PRs, discussions etc./ that are relevant to me today despite no specific event (with notification) having showed up today.

# ADDITIONAL:

Sometimes, I contribute to FOSS in my spare time. Sometimes, I contribute to FOSS profesionally, or just have feedback to various projects.

I would like to find those things again, but 90+% of my notifications are from one single organization, my employer.

You can filter by organization, so `org:employer` works, but you cannot *exclude* by organization, so `-org:employer` does *not* work.

You can group by repository, but 90+% of the repositories I touch are at my employer, so that doesn't solve the problem either.

Subscriptions (https://github.com/notifications/subscriptions) are also a good source for re-finding these things, but subscriptions are even harder to filter

# POSSIBLE SOLUTIONS:

The gold solution for me would be a page that has a list of every notification, subscription, PR and discussion that I have either touched or been mentioned in, with a search functionality that includes exclude-filters, and allows ordering by date/organization/repo.

**Comment by dakshithadissanayaka** [comment]
Given your requirements and the limitations of the current GitHub interface, it seems like a comprehensive solution to filter, search, and organize your notifications and discussions would be beneficial. While GitHub's current features may not cover all your needs, there are some workarounds and tools you can use to improve your experience:

1. **Use GitHub Search Bar:**
   - GitHub's search bar can be quite powerful. You can use it to search for specific terms, including your username, repository names, or specific keywords related to discussions or pull requests you've been involved in.

     Example:
     ```
     author:yourusername is:issue is:open
     ```

2. **Email Notifications:**
   - If you receive email notifications for your GitHub activities, you can use your email client's search and filtering capabilities to organize and find relevant discussions.

3. **Custom Labels in Issues/PRs:**
   - Encourage the use of custom labels in your issues or pull requests. This allows you to categorize and filter discussions based on specific criteria. You can create labels like "Personal Contribution," "FOSS," etc.

4. **Third-Party Tools:**
   - Consider using third-party tools that integrate with GitHub and provide advanced search and filter functionalities. Some popular tools include Octobox and Hound. These tools often provide more advanced features than the GitHub interface.

5. **Feature Request:**
   - Submit a feature request to GitHub for improved notification and search functionalities. GitHub is actively developed, and user feedback can influence future updates and features.

6. **GitHub API:**
   - If you are comfortable with scripting and programming, you can use the GitHub API to create custom scripts or tools that meet your specific needs. This allows you to fetch and filter notifications programmatically.

Remember that GitHub's features and interface might evolve, so periodically check for updates or new features that could address your requirements. Providing feedback to GitHub through their official channels is also a valuable way to influence the development of the platform.
